chrome/content/test/test_loader.html

```diff
@@ -9,6 +9,11 @@
     <script type="text/javascript" src="../vendor/jasmine/lib/jasmine-2.0.0/boot.js"></script>
     <script type="text/javascript" src="../vendor/jasmine/reporters/terminal_reporter.js"></script>
     <!-- Tests scripts -->
-    <script type="text/javascript" src="./tests.js"></script>
+    <script type="text/javascript" src="./globals.js"></script>
```

Comment:
Have you written up a few paragraphs on how the test loading works? I am not sure where this test loader is loaded from/what technologies it uses, etc. A big header in a source file or a README would be helpful.